docs/CONTRIBUTING.adoc

```diff
@@ -179,3 +179,38 @@ To submit a pull request:
 . Click the _Create pull request_ button.
 
 The pull request will be built in link:https://ci.jenkins.io[ci.jenkins.io].
+
+== Releases
+
+The releases can be generated by using the `release.sh` script at the root of the repository.
+This scripts uses `git`, `gh`, `jq` and `mvn` to generate the release.
+The script also requires Bash version to be greater than 4.
+Make sure you have all those tools on your system.
+
+You can run the script like this:
+
+[source bash]
+----
+bash release.sh
+----
+
+WARNING: to be able to perform a release, you **must** have write permission on the project repository.
+
+The script will pick the version from the GitHub Release in draft which is prepared by `release-drafter`.
+
+You can pass a version as the first argument of the script, but only if you know exactly what you are doing.
+Then, run the script like:
+
+[source bash]
+----
+bash release.sh 9.9.9
+----
+
+The project doesn't publish Java binary.
```

Comment:
```suggestion
The project doesn't publish a Java binary.
```
Not sure if it needs the `a` here, but suggesting in case it was missed.

Reply:
It sounds better with it